modules/aleph/includes/aleph.reservation.inc

```diff
@@ -37,7 +111,28 @@ function aleph_reservation_reservation_deletion_enabled() {
   return FALSE;
 }
 
-function aleph_reservation_branch_name() {
+/**
+ * Return a branch name for a given branch id.
+ *
+ * @param string $branch_id
+ *   The identifier for a given pick-up branch (e.g. BBAAA).
+ *
+ * @return string
+ *   The human name for the id or NULL if one do not exist.
+ */
+function aleph_reservation_branch_name($branch_id) {
+  if (strlen($branch_id) > 5) {
+    return $branch_id;
+  }
+
+  $branches = array(
```

Comment:
I do hope this is just a placeholder until some branch fetching code is written.

In which case it's missing a `TODO` comment.

Reply:
Yup, todo added, @xendk. :)